### DISCUSSION TITLE
Looks like it won't start.

### DISCUSSION THREAD

**Discussion by Lltnice** [Genymobile/feedback#84] [Genymotion Desktop] [answered]
![}%B@9)R AP1$O7IYATFKGZA](https://github.com/Genymobile/feedback/assets/157012925/f49e0a13-add2-45f9-bdae-d1d203c7a33d)
After startup, it will always show startup, but there is no follow-up, I want to know what is the problem

**Comment by XxMrNobodyxX** [comment]
Yeah same here...First time coming back to Genymotion in 6 months and I've never seen it this bad. 

![image](https://github.com/Genymobile/feedback/assets/58698256/35aba3c1-253e-41ea-8e36-893e9dd92965)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @Lltnice @Captain404 ,
This board is not suitable for technical support requests. Please visit our help center at https://support.genymotion.com/hc/en-us to find a solution to your issue.
Thank you for your understanding.
